CHANGELOG.md

```diff
@@ -8,6 +8,76 @@ This project uses [*towncrier*](https://towncrier.readthedocs.io/) and the chang
 
 <!-- towncrier release notes start -->
 
+## [3.113.0](https://github.com/metalbear-co/mirrord/tree/3.113.0) - 2024-08-12
+
+
+### Added
+
+- Add new api to run mirrord inside container
+
+  ```
+  mirrord container [options] -- <docker/podman> run ...
+  ```
+
+  Because we need to run internal proxy process on the same network as the
+  process loaded with `mirrord-layer`, to keep config and kubernetes
+  comparability the communication to mirrord agent is made via external proxy
+  that will run on the host machine.
+  ```
```

Comment:
I just have a dilemma, I like the graphic but. not sure should be in changelog

Reply:
Nah, I think the graphic is cool. Why not?

Reply:
It's just Iv'e never seen one in any changelog

Reply:
@eyalb181 wdyt?

Reply:
Let's keep it in, it's attention grabbing